src/pybind/mgr/dashboard/settings.py

```diff
@@ -62,6 +62,27 @@ class Options(object):
     USER_PWD_EXPIRATION_WARNING_1 = (10, int)
     USER_PWD_EXPIRATION_WARNING_2 = (5, int)
 
+    # Password policy
+    PWD_POLICY_ENABLED = (True, bool)
+    # Individual checks
+    PWD_POLICY_CHECK_LENGTH_ENABLED = (True, bool)
```

Comment:
I'm wondering why these settings are not going to be shown as checkboxes (e.g. as it is the case for 'FEATURE_TOGGLE_rgw'). I think this would be more convenient.

The issue might not relate to the PR directly as it's the same for GRAFANA_UPDATE_DASHBOARDS or GRAFANA_API_SSL_VERIFY.

![Screenshot_2020-01-23_14-50-10](https://user-images.githubusercontent.com/8761082/72990293-317de100-3df0-11ea-8bf1-dd4d736e7e78.png)

Reply:
I think it's because the type of the settings defined in `settings.py` is not  identified correctly. I'll fix that if possible in a separate PR because this is not related to this one.

Reply:
https://tracker.ceph.com/issues/43769